branchLayout/impl/src/main/java/com/virtuslab/branchlayout/impl/readwrite/BranchLayoutReader.java

```diff
@@ -171,7 +172,7 @@ private Array<Tuple2<Integer, Integer>> parseToArrayRepresentation(Path path, In
 
     if (indent % indentSpec.getIndentWidth() != 0) {
       throw new BranchLayoutException(lineNumber + 1,
-          "Levels of indentation are not matching in branch layout file (${path.toAbsolutePath()}): " +
+          "Levels of indentation are not matching in branch layout file: " +
```

Comment:
I am wondering if it is worth to provide the path here 🤔 a specific path could be useful, wdyt?

Reply:
probably we can still determine the specific path based on the other logs, right?

Reply:
Let's leave as is, it shouldn't introduce much mess